lib/Tool/Requirements.php

```diff
@@ -615,6 +616,16 @@ public static function checkPhp()
             'state' => class_exists('Imagick') ? Check::STATE_OK : Check::STATE_WARNING,
         ]);
 
+        if(class_exists('Imagick')) {
+            $imageMagickLcmsDelegateInstalledProcess = Process::fromShellCommandline('convert -list configure | grep DELEGATES');
```

Comment:
We should use `\Pimcore\Tool\Console::getExecutable()` to determine the path to `convert`.

Reply:
I'd prefer to not use `grep` but pure PHP.

Reply:
Done.